### DISCUSSION TITLE
Klipper shutdown when attempting to test resonance

### DISCUSSION THREAD

**Discussion by MakManIfest** [Guilouz/Creality-Helper-Script-Wiki#501] [Issues]
As title states.
K1 Max.
1.3.3.8
Had issues installing the improved shaper calibration. 
1st time I got an error about my camera hardware and said I couldn't install.
2nd time the .cfg files got modified but the helper script files were missing.
3rd time everything seemed to install but I get this error.

![K1max_error2](https://github.com/Guilouz/Creality-K1-Series/assets/89977456/f536f86f-7b3c-4642-9689-7860cd00e2be)

Dunno what's going on. Any help is appreciated.

**Comment by MakManIfest** [comment]
I've reset to factory and down graded to 1.3.3.5, re rooted and re installed via the helper script.

Other than having issues with certificates causing moonraker to partially install and other packages to fail. Restarting the printer resolved these issues. Then updating moonraker in mainsail to fix the broken moonraker install.

Everything seems to be working as it should be now.

Some gremlins in 1.3.3.8 me thinks.

**Reply by mxfi** [reply]
This might have to do with the script overloading the memory if the values are too far. I found that when I have my springs in the toolhead loose or replace it with too much pressure and there's binding on the x axis, I also get a crash with he same error right after it finishes vibrating

**Comment by MateuszKrawczuk** [comment]
I also occurred same error on K1C.

**Comment by B-Man1** [comment]
I also get crashing when running resonance calibration.  Sometimes after x is done, sometimes after y is done, sometimes it does not crash.  All times is very slow and resource intensive.

I also have moonraker warning issues that I have not yet been able to resolve
